projects/examples/src/components/quick-search/quick-search-sync-async.example.component.html

```diff
@@ -16,6 +16,12 @@
         description="The text to be shown in the quick search input field when it is empty"
     >
     </vcd-form-input>
+    <vcd-form-checkbox
```

Comment:
Please create a separate example including some explanation for the feature. Each example should show one feature. If an example does want to show interacting features, it should be named accordingly. This example is supposed to show the fact that a provider can be sync or async

Reply:
done